Marlin/language_da.h

```diff
@@ -45,12 +47,12 @@
 #define MSG_AUTO_HOME_Y                     "Home Y"
 #define MSG_AUTO_HOME_Z                     "Home Z"
 #define MSG_LEVEL_BED_HOMING                "Homing XYZ"
-#define MSG_LEVEL_BED_WAITING               "Klik når du er klar"
+#define MSG_LEVEL_BED_WAITING               "Tryk for at starte bed level"
 #define MSG_LEVEL_BED_NEXT_POINT            "Næste punkt"
 #define MSG_LEVEL_BED_DONE                  "Bed level er færdig!"
 #define MSG_LEVEL_BED_CANCEL                "Annuller bed level"
-#define MSG_SET_HOME_OFFSETS                "Sæt forsk. af home"
-#define MSG_HOME_OFFSETS_APPLIED            "Forsk. er nu aktiv"
+#define MSG_SET_HOME_OFFSETS                "Sæt forskyding af home"
+#define MSG_HOME_OFFSETS_APPLIED            "Forskydninger af home pos. er tilføjet"
```

Comment:
These strings are for use on LCD controllers, which have a nominal limit of 20 characters. Translators are asked to strive for 18 or fewer characters, if possible, and 16 or fewer characters ideally.